script.js

```diff
@@ -0,0 +1,135 @@
+window.addEventListener('load', init)
+
+var addCart = [    
+    {Name:'Котеров, Симдянов: PHP 7', cost:1080 , cart:'Add to Cart' },
```

Comment:
Не используйте заглавную букву. Это зарезервировано для классов
```suggestion
    {name:'Котеров, Симдянов: PHP 7', cost:1080 , cart:'Add to Cart' },
```